prawcore/requestor.py

```diff
@@ -47,9 +47,11 @@ def close(self):
         """Call close on the underlying session."""
         return self._http.close()
 
-    def request(self, *args, timeout=TIMEOUT, **kwargs):
+    def request(self, *args, timeout=None, **kwargs):
         """Issue the HTTP request capturing any errors that may occur."""
         try:
-            return self._http.request(*args, timeout=timeout, **kwargs)
+            return self._http.request(
```

Comment:
Why is this changing?

Reply:
The parent requestor functions have a default of None, so I decided to change this as well.

Reply:
The problem with doing it this way is that `None` then cannot be passed in to disable our default timeout.